chainercv/evaluations/eval_semantic_segmentation.py

```diff
@@ -59,15 +59,13 @@ def eval_semantic_segmentation(pred_label, gt_label, n_class):
     Args:
         pred_label (array): An integer array of image containing
```

Comment:
Can we assume that the sizes of input images are same? For example, the sizes of images from `VOCSemanticSegmentationDataset` are different.